src/components/screens/ExperimentView.vue

```diff
@@ -110,14 +109,14 @@
                 {{ targetGene.targetAccession.accession }}
               </div>
 
+              <div v-if="targetGene.targetSequence.taxonomy?.url"><strong>Taxonomy ID:</strong> 
+                &nbsp;<a :href="`${targetGene.targetSequence.taxonomy.url}`" target="blank">{{targetGene.targetSequence.taxonomy.taxId}}</a> 
+                </div>
               <div v-if="targetGene.targetSequence?.sequence" style="word-break: break-word">
-                <div v-if="targetGene.targetSequence.reference?.organismName"><strong>Organism:</strong>
-                  {{ targetGene.targetSequence.reference.organismName }}</div>
-                <div v-if="targetGene.targetSequence.reference?.shortName"><strong>Reference genome:</strong>
-                  {{ targetGene.targetSequence.reference.shortName }}</div>
-                <div v-if="targetGene.targetSequence.reference?.id"><strong>Genome ID:</strong>
```

Comment:
is there anything that should replace this Genome ID/Target ID display in the new version?

Reply:
Reference_genome won't exist anymore. They're replaced by taxonomy ID. 
BUT, Target ID is wrong deleted! I'll add it back.